src/main/java/com/puppycrawl/tools/checkstyle/meta/MetadataGeneratorUtil.java

```diff
@@ -46,11 +45,13 @@ private MetadataGeneratorUtil() {
     /**
      * Generate metadata from the module source files available in the input argument path.
      *
-     * @param args arguments
+     * @param path arguments
+     * @param moduleFolders List of folders to check
      * @throws IOException ioException
      * @throws CheckstyleException checkstyleException
      */
-    public static void generate(String... args) throws IOException, CheckstyleException {
+    public static void generate(List<String> moduleFolders, String path)
```

Comment:
it may be easier to make it 
`public static void generate(String path, String... moduleFolders)` to not create some lists

Reply:
It  will be even more logical while reading a code

Reply:
Changes done